sdk/python/pulumi_azuread/__init__.py

```diff
@@ -21,3 +21,9 @@
 from .service_principal_certificate import *
 from .service_principal_password import *
 from .user import *
+from ._inputs import *
+from . import outputs
+
+# Make subpackages available:
+from . import (
+)
```

Comment:
Oops. Ideally we wouldn't emit this if there aren't any subpackages. Need to check to see if having this here is benign (to be cleaned up in a subsequent codegen fix) or if it needs to be fixed now. Will check.

Reply:
Yep, need to fix the provider codegen. Working on a fix.

Reply:
https://github.com/pulumi/pulumi/pull/5195

Reply:
We'll need to update `go.mod` require for `github.com/pulumi/pulumi/sdk/v2` in this PR to reference this commit: https://github.com/pulumi/pulumi/commit/ca7536e17f87687a3facdcb2e3aca0fa8017ebd4